### ISSUE TITLE
Boleto próprio

### ISSUE THREAD

**Issue by NaldoAlmeida** [eduardokum/laravel-boleto#763]
Seria interessante gerar um boleto para pagamento na própria empresa geradora, como se ela fosse o banco. Em vez de aparecer o código de barras, seria um texto "Pagar no local xyz..", mas com toda estrutura do boleto ou carnê.

Esse recurso já existe?

**Comment by eduardokum** [comment]
não, teria que criar uma classe no seu proprio projeto estendendo a AbstractBoleto e fazer os calculos de digito e confeccao da linha digitavel conforme sua necessidade
